docs/content/user-guide/en/transport/azure-service-bus.md

```diff
@@ -109,3 +110,30 @@ c.UseAzureServiceBus(asb =>
         };
 });
 ```
+
+#### SQL Filters
```

Comment:
Just a minor glitch: shouldn't it be custom consumers instead of SQL Filters? That section makes more sense than the property since this is the API that is going to be public.